syft/pkg/cataloger/golang/licenses_test.go

```diff
@@ -26,6 +28,8 @@ func Test_LocalLicenseSearch(t *testing.T) {
 	loc2 := file.NewLocation("github.com/!cap!o!r!g/!cap!project@v4.111.5/LICENSE.txt")
 	loc3 := file.NewLocation("github.com/someorg/strangelicense@v1.2.3/LiCeNsE.tXt")
 
+	licenseScanner := licenses.StaticScanner()
```

Comment:
This calls a function that loudly documents that it's just for testing. Is there a reason we don't want to make the real license scanner here via `NewDefaultScanner`? Does doing that make the unit tests too slow?

Reply:
> Does doing that make the unit tests too slow

it's hard to quantify "too slow", but yes, we'd be instantiating a license scanner for every test case this is referenced, which was more than 1 (looks to affect ~33 cases).

Reply:
disregard, this wasn't reproducible. I'll remove the static scanner